### PR TITLE
Promote PodShareProcessNamespace to stable

### PR DESCRIPTION
**Pull request by verb**
Updates website for kubernetes/kubernetes#84356 to promote PodShareProcessNamespace to GA.

### CODE REVIEW

**Comment by verb** [comment]
do-not-merge until kubernetes/kubernetes#84356

/hold

File: content/en/docs/tasks/configure-pod-container/share-process-namespace.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ weight: 160
 
 {{% capture overview %}}
 
-{{< feature-state state="beta" >}}
+{{< feature-state state="stable" >}}
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/share-process-namespace.md]:
```suggestion
{{< feature-state state="stable" for_k8s_version="v1.17" >}}
```

File: content/en/docs/tasks/configure-pod-container/share-process-namespace.md [hunk 2]
```diff
@@ -25,10 +25,7 @@ include debugging utilities like a shell.
 
 {{% capture prerequisites %}}
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
-Process Namespace Sharing is a **beta** feature that is enabled by default. It
-may be disabled by setting `--feature-gates=PodShareProcessNamespace=false`.
+{{< include "task-tutorial-prereqs.md" >}}
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/share-process-namespace.md]: Why omit the `{{< version-check >}}` shortcode?

Comment (verb) [reply content/en/docs/tasks/configure-pod-container/share-process-namespace.md]: Just assumed it was on the list of things to remove for GA. I can add it back.

The generated text is pretty misleading, though:

> Your Kubernetes server must be at or later than version v1.10. To check the version, enter kubectl version.

In fact if your Kubernetes server is at version v1.10 and you haven't specifically enabled the feature, then the task will fail.

**Comment by verb** [comment]
The k/k PR now has approval, remove holding

/hold cancel

**Comment by daminisatya** [comment]
Just a reminder about the last Docs deadline - 22nd Nov, by which this PR needs to be merged!

**Comment by daminisatya** [comment]
@verb @zacharysarah How can we make sure we reach out to the right reviewer for the tech review?

**Comment by verb** [comment]
> @verb @zacharysarah How can we make sure we reach out to the right reviewer for the tech review?

@daminisatya I'm just waiting for approval. My previous comment about versioning isn't a blocker for this release.

@chenopis This PR is a simple s/beta/stable/ and should be a quick review.

**Comment by zacharysarah** [comment]
@daminisatya Can @verb recommend a reviewer from the relevant SIG?

/unassign @chenopis 
/assign @sftim

**Pull request merged**